src/main/scala/io/viash/engines/requirements/RRequirements.scala

```diff
@@ -101,7 +101,7 @@ case class RRequirements(
     val prefix = if (warnings_as_errors) "options(warn = 2); " else ""
 
     def runRCode(code: String): String = {
-      s"""Rscript -e "${prefix}${code.replaceAll("\"", "\\\"")}""""
+      s"""Rscript -e '${prefix}${code}'"""
```

Comment:
switching to `'` instead of `"` makes sense! 

Should we not escape `'` in the code?

```suggestion
      s"""Rscript -e '${prefix}${code.replaceAll("'", "'\"'\"'"}'"""